src/NexusMods.App.UI/WorkspaceSystem/Workspace/IWorkspaceViewModel.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.ObjectModel;
+using System.Reactive;
+using Avalonia;
+using ReactiveUI;
+
+namespace NexusMods.App.UI.WorkspaceSystem;
+
+public interface IWorkspaceViewModel : IViewModelInterface
```

Comment:
Here as well, a comment describing what a Workspace is and its purpose. We had some naming issues on these so having some clarification in the code is important I think.

Reply:
I'll add documentation once I add tabs to the system.